### DISCUSSION TITLE
Includes Sumatra PDF Ghostscript?

### DISCUSSION THREAD

**Discussion by SteffenGeller** [sumatrapdfreader/sumatrapdf#3613] [Documentation]
We are using Sumatra PDF version 2.5.3. Now we have a security warning for Sumatra PDF from our SOC team due to Ghostscript. Does Sumatra PDF include Ghostscript? Is there a security update for this?

**Comment by GitHubRulesOK** [comment]
SumatraPDF does not include Ghostscript as they are two seperate Artifex applications HOWEVER it will for EPS OR PS viewing use any independently installed GhostScript to convert the files into PDF, so uninstall your GhostScript and SumatraPDF will not recognise it as available.

2.5.3 could only run older GhostScripts ( [it fails around 9.54](https://github.com/sumatrapdfreader/sumatrapdf/issues/1923#top)) and now we are beyond version 10 ( 10.01.2 )
SumatraPDF releases after May 27, 2021 (3.3+) do not use the same older GhostScript version upto 9 calling method but if older ghostscript is installed may attempt to use that for ps and eps files.

discussion also at https://github.com/sumatrapdfreader/sumatrapdf/issues/3577
